docs/pages/eas-update/rollouts.mdx

```diff
@@ -1,32 +1,76 @@
 ---
 title: Rollouts
-description: Incrementally deploy updates on a new branch to a channel.
+description: Incrementally deploy updates to your users.
 ---
 
+import { Collapsible } from '~/ui/components/Collapsible';
 import { Terminal } from '~/ui/components/Snippet';
 
-> **info** Available from SDK 50 using [`expo-updates`](/versions/latest/sdk/updates/) 0.23.0 or above.
+A rollout allows you to roll out a change to a portion of your users in order to catch bugs or other issues before releasing that change to all your users.
 
-Rollouts incrementally deploy updates on a new branch to a specific channel. With rollouts, you can roll out updates from one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to our production environment.
+EAS provides two rollout mechanisms depending on your use case:
```

Comment:
```suggestion
EAS provides **pre-update and branch-based rollout mechanisms** to suit your use case.
```

Instead of using a colon here, let's add those mechanisms here.